README.md

```diff
@@ -28,6 +28,21 @@ def handler(event, context):
 
 For a full example see the [example folder](./example).
 
+## Usage in a Cumulus Deployment
+During deployment, Cumulus will automatically obtain and inject the [Cumulus Message Adapter](https://github.com/cumulus-nasa/cumulus-message-adapter) zip into the compiled code and create a zip file to be deployed to Lambda.
```

Comment:
I had the same suggestion for Lauren earlier.  The fact that the message adapter is a zip is an implementation detail that task authors don't really need to care about.  That could be omitted.

Reply:
I need to make updates to the README anyway, seth, so you can leave this

Reply:
:+1: